PhiferTristan/kneel-server
# Orders View

Added methods to OrdersView class to handle POST, PUT, and DELETE requests.

## Steps to Test

1. Pull branch "tp-orders-view" and switch to it
2. Launch debugger to connect to local host
3. Open Postman
4. Send a DELETE or PUT request to the path ending with `/orders/n` and verify data has been deleted or updated by refreshing 
    your tables in the kneeldiamonds.sqlite3 file.
5. Send a POST request to the path ending with `/orders` including a body of what raw data you what sent to the database with a format like so, "n" being which id you want to choose, and verifying your data by refreshing your tables in kneeldiamonds.sqlite3:

```
{
    "metal_id": n,
    "size_id": n,
    "style_id": n
}
```
